package.json

```diff
@@ -199,6 +199,9 @@
 		"source-map": "0.7.3",
 		"source-map-support": "0.5.16",
 		"store": "2.0.12",
+		"@storybook/addon-actions": "5.2.8",
+		"@storybook/preset-scss": "1.0.2",
+		"@storybook/react": "5.2.8",
```

Comment:
These should be `devDependencies` instead of `dependencies`, is that right? The Calypso app itself doesn't need them to run.

The distinction will become important when we merge #38190. Then the Calypso app dependencies will be in `client/package.json` and the global dev dependencies will remain in root `package.json`. In that PR, I had to move several incorrectly placed dev dependencies to make the build work.